packages/orchestration/src/orchestration-api.ts

```diff
@@ -124,6 +122,24 @@ export interface Orchestrator {
     /** the Denom for the underlying asset on its issuer chain */
     baseDenom: Denom;
   };
+
+  /**
+   * Get denom on a holding chain.
+   *
+   * @param baseDenom denom on issuing chain
+   * @param base issuing chain
+   * @param chain holding chain - connection to base chain must be regitered in chainHub
+   * @returns denom on holding chain
+   */
+  getDenomOn: <
+    HoldingChain extends keyof KnownChains,
```

Comment:
not strictly necessary that it's a known chain. but we can tackle that when we have more use of contract-specified chains

Reply:
I followed the structure of `getBrandInfo()`. I guess the comment applies to both?